app/lib/activitypub/tag_manager.rb

```diff
@@ -13,7 +13,7 @@ class ActivityPub::TagManager
   }.freeze
 
   def public_collection?(uri)
-    uri == COLLECTIONS[:public] || uri == 'as:Public' || uri == 'Public'
+    uri == COLLECTIONS[:public] || ['as:Public', 'Public'].include?(uri)
```

Comment:
```suggestion
    uri == COLLECTIONS[:public] || %w(as:Public Public).include?(uri)
```

Reply:
I re-ran the format after applying, just to see if Rubocop would complain, but it seems happy with this array format too